modules/flowable-cmmn-api/src/main/java/org/flowable/cmmn/api/CmmnManagementService.java

```diff
@@ -318,4 +319,31 @@
      */
     CmmnExternalWorkerTransitionBuilder createCmmnExternalWorkerTransitionBuilder(String externalJobId, String workerId);
 
+    /**
+     * Create a {@link ChangeTenantIdBuilder} that can be used to change the tenant id of the case instances
+     * and all the related instances, including:
+     * <ul>
+     *    <li>CaseInstances</li>
+     *    <li>HistoricCaseInstances</li>
+     *    <li>PlanItemInstances</li>
+     *    <li>HistoricPlanItemInstances</li>
+     *    <li>MilestoneInstances</li>
+     *    <li>HistoricMilestoneInstances</li>
+     *    <li>Tasks</li>
+     *    <li>HistoricTaskInstances</li>
+     *    <li>HistoricTaskLogEntries</li>
+     *    <li>EventSubscriptions</li>
+     *    <li>Jobs</li>
+     *    <li>TimerJobs</li>
+     *    <li>SuspendedJobs</li>
+     *    <li>DeadLetterJobs</li>
+     *    <li>HistoryJobs</li>
+     *    <li>ExternalWorkerJobs</li>
+     * </ul>
+     * <p>
+     * You must provide the source tenant id and the destination tenant id. All instances from the source tenant id in the CMMN scope
+     * will be changed to the target tenant id.
+     * 
+     */    ChangeTenantIdBuilder createChangeTenantIdBuilder(String fromTenantId, String toTenantId);
```

Comment:
Please format this correctly. The Java code should be below the comment line. This is meant for all management services

Reply:
fixed